changelog.d/564.bugfix

```diff
@@ -0,0 +1 @@
+Make expire, pexpire, expireat and persist return the redis client value
```

Comment:
I would probably characterize this as a new feature for this library rather than a bugfix.

Also, I'm not sure, but it's possible towncrier may handle a feature and bugfix being the same issue number, but I would personally expect a PR to have one or the other defined (though I can imagine a case where that's not true)

Reply:
Tested towncrier locally, it handles two files for the same issue.
I added the pexire_at at request of @WisdomPill :)